libcxx.mixin

```diff
@@ -0,0 +1,8 @@
+build:
+  libcxx:
+    cmake-args:
+      - "-DCMAKE_C_COMPILER=clang-6.0"
+      - "-DCMAKE_CXX_COMPILER=clang++-6.0"
```

Comment:
The last hard coded information is the `clang` version. Why should version 6.0 be chosen? That looks like an arbitrary decision driven by a specific platform.

Reply:
Good point - clang6.0 is the default on bionic and it's not symlinked by default, but users can `update-alternatives` to get `clang` and `clang++`. I have removed the versions.